pkg/registry/core/rest/storage_core.go

```diff
@@ -340,7 +341,7 @@ type componentStatusStorage struct {
 func (s componentStatusStorage) serversToValidate() map[string]*componentstatus.Server {
 	serversToValidate := map[string]*componentstatus.Server{
 		"controller-manager": {Addr: "127.0.0.1", Port: ports.InsecureKubeControllerManagerPort, Path: "/healthz"},
-		"scheduler":          {Addr: "127.0.0.1", Port: ports.InsecureSchedulerPort, Path: "/healthz"},
+		"scheduler":          {Addr: "127.0.0.1", Port: kubeschedulerconfig.InsecureSchedulerPort, Path: "/healthz"},
```

Comment:
add a comment indicating this is fragile and assumes the default port is being used